src/lib/executors/Node.ts

```diff
@@ -269,6 +290,14 @@ export default class Node extends Executor<Events, Config, NodePlugins> {
 	}
 
 	protected _beforeRun() {
+		this._coverageFiles.forEach(filename => {
```

Comment:
This adds quite a startup delay, and ends up instrumenting any files being tested twice. Could we do it at the end of unit tests, and only for files that didn't already have coverage data?